docs/migration/migrate_8_0.asciidoc

```diff
@@ -100,7 +100,7 @@ See https://github.com/elastic/kibana/pull/87939 for more details.
 
 [float]
 ==== Logging destination is specified by the appender
-*Details:* Previously log destination would be `stdout` and could be changed to `file` using `logging.dest`. With the new logging configuration, you can specify the destination using {kibana-ref}/logging-service.html#logging-appenders[appenders]. 
```

Comment:
Link destination removed.

Reply:
Are you planning on adding these links back too after merging the PR?

Reply:
Yes, although we should actually keep everything else in the `*Details:*` section.
```suggestion
*Details:* Previously log destination would be `stdout` and could be changed to `file` using `logging.dest`. With the new logging configuration, you can specify the destination using the file-type appender.
```

Reply:
yes